### ISSUE TITLE
re-enable gio and gtk in stackage

### ISSUE THREAD

**Issue by debug-ito** [gtk2hs/gtk2hs#228]
Currently gio and gtk are not in stackage because they were not compatible with Cabal-2.0 (fpco/stackage#3048). Now they are OK with Cabal-2.0, could you re-enable them in stackage?

**Comment by debug-ito** [comment]
If it's ok, I'd love to make pull-request to stackage to re-enable gio and gtk.

**Comment by cohomology** [comment]
Hi,

is this the reason why gtk is not available on stackage for resolvers >= 10?

Is it possible to help here somehow?

Best regards,
Kilian.

**Comment by debug-ito** [comment]
I think this is the reason.

As far as I confirmed, the latest packages of gio and gtk are already buildable. They just need to be enabled in stackage.

**Comment by cohomology** [comment]
Ok, so I guess this is something which can't be fixed _here_, but on must be fixed on [Stackage](https://github.com/fpco/stackage)?

**Comment by debug-ito** [comment]
Exactly.

Technically, anybody can send a pull-request to the stackage repository to re-enable gio and gtk. I wonder if it's ok for maintainers of gtk2hs. I guess it's probably ok, though.

**Comment by cohomology** [comment]
Yes. They did not disable themselves, so they should be okay to be reenabled. Can you do that?

**Comment by debug-ito** [comment]
OK, I will.

**Comment by debug-ito** [comment]
Hmm, looks like ghc-8.4.1 is just introduced in nightly-2018-03-13. I'm gonna check if gio and gtk can build later..

**Comment by debug-ito** [comment]
Looks like it's impossible to build gio and gtk with current stackage nightly (nightly-2018-04-05), because it's incompatible with Cabal-2.2. Besides, gtk2hs-buildtools probably has a problem, too (#242).

**Comment by schrammc** [comment]
Is there any chance that cairo could be put on stackage again? 

Cabal compatibility seems to no longer be a problem.

**Comment by debug-ito** [comment]
@hamishmack any idea?
